src/components/join/JoinForm.tsx

```
@@ -0,0 +1,128 @@
+        if (!nickName) return;
+
+        try {
+            const res = await checkNickName(nickName);
+            if (res.available) {
+                setNicknameAvailable(true);
+                setNickNameMessage('사용 가능한 닉네임입니다!');
+            } else {
+                setNicknameAvailable(false);
+                setNickNameMessage('이미 사용 중인 닉네임입니다.');
+            }
+        } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)